samples/ui-kit-place-details-compact/style.css

```diff
@@ -33,46 +33,28 @@ gmp-place-details-compact {
   --gmp-mat-font-family: Google Sans Text, sans-serif;
   /* Defines the style for medium body text (e.g., address, descriptions) */
   --gmp-mat-font-body-medium: normal 400 0.875em/1.25em var(--gmp-mat-font-family, 'Google Sans Text');
-  
+
   width: 360px;
   border: none;
-  overflow-y: hidden;
-}
-
-.gm-style-iw button {
-    display: none !important;
-}
-
-.gm-style-iw-c {
-    padding: 0 !important;
-    margin: 0 !important;
-}
-
-.gm-style-iw-d {
-    padding: 0 !important;
-    margin: 0 !important; /* Sometimes margin can also contribute to unwanted space */
-    overflow: visible !important; /* If you don't want scrollbars, but be careful with content overflow */
+  padding: 0;
+  margin: 0;
+  position: relative;
+  transform: translate(0, calc(- 20px));
```

Comment:
There is an issue with this style (was it supposed to be -20px rather than an infix?)

▲ [WARNING] "-" can only be used as an infix operator, not a prefix operator [invalid-calc]

    <stdin>:42:31:
      42 │   transform: translate(0, calc(- 20px));
         ╵                                ^